charts/webhook-proxy/README.md

```diff
@@ -0,0 +1,11 @@
+To deploy webhook-proxy against your console edit `values.yaml` to ensure that `configMap.host` points to your console instance:
+
+```yaml
+# Config map with NGINX configuration.
+configMap:
+  # The name of the config map to use.
+  # If not set, a name is generated using the fullname template
+  name: ""
```

Comment:
i'd drop the name/annotations part here, just set the host

Reply:
@michaeljguarino done. If there is anything else you would like to see there let me know. At the moment it's basically standard Helm deployment with config map on top of it.